dynawo/sources/Models/Modelica/PreassembledModels/CMakeLists.txt

```diff
@@ -85,7 +85,8 @@ set(MODELICA_PREASSEMBLED_MODELS
     StaticVarCompensatorPVRemote.xml
     Step.xml
     TapChangerAutomaton.xml
-    TapChangerBlockingArea.xml
+    TapChangerBlockingArea1.xml
+    TapChangerBlockingArea2.xml
```

Comment:
I'm not very fan of area. What do you think of TapChangerBlockingAutomaton1 or TapChangerBlocking1?